6/fundamentals/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM debian:stretch-slim as mozjpeg
 ENV VERSION 3.3.1
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends autoconf automake checkinstall libpng-dev libtool make nasm pkg-config && \
+    apt list -a ffmpeg && \
+    apt-get install -y --no-install-recommends autoconf automake checkinstall libpng-dev libtool make nasm pkg-config ffmpeg=7:3.2.15-0  && \
```

Comment:
Why installing `ffmpeg`? It won't last in fundamentals. Is it still needed?

Reply:
It already comes pre-installed by `mvhirsch/thumbor:slim` (in last stage). This change won't affect anything, but making build durations longer.

Please remove it from here.